language/ast.py

```diff
@@ -37,20 +37,21 @@ class ID(AbstractID):
 
 
 @abstract
-class Type(RFLXNode):
+class Declaration(RFLXNode):
     """
-    Base class for type declarations (enums, integers, messages, arrays, derivations)
+    Base class for declarations (types, refinements, sessions)
     """
 
 
 @abstract
 class TypeDef(RFLXNode):
     """
-    Base class for type definitions
+    Base class for type definitions (integers, abstract messages, type derivations, arrays,
+    enumerations)
```

Comment:
Why "abstract messages" and not just "messages"?

Reply:
Changed in dce744528c85e4b63b5b86f2fd40b00ddd8233a9.